chef_master/source/api_omnitruck.rst

```diff
@@ -115,7 +115,7 @@ Omnitruck accepts the following platforms:
    * - Microsoft Windows
      - ``windows``
      - ``x86_64``
-     - ``2016``, ``2012r2``, ``2012``, ``8``
+     - ``2019``, ``2016``, ``2012r2``, ``2012``, ``8``
```

Comment:
Why do we have an ascending order in one case, but descending in the other?

Reply:
No clue

Reply:
Fixed